modules/helm/helm.mk

```
@@ -187,3 +190,19 @@ verify-helm-kubeconform: $(helm_chart_archive) | $(NEEDS_KUBECONFORM)
 		-strict
 
 shared_verify_targets_dirty += verify-helm-kubeconform
+
+.PHONY: helm-diff
+helm-diff: $(helm_chart_archive) | $(NEEDS_HELM) 
+	@if [ -z "$(helm_chart_old_version)" ]; then \
+		echo "Usage: make helm-diff helm_chart_old_version=<version>"; \
+		exit 1; \
+	fi
+
+	$(eval export TMP_OLD=$$(mktemp -d))
+	$(eval export TMP_NEW=$$(mktemp -d))
+	$(eval export OUTPUT_DIR=$$(mktemp -d))
```

Oops, i might have given you the wrong command, I'm now getting:

```
$ make helm-diff
make/_shared/helm///helm.mk:196: warning: undefined variable `mktemp -d'
make/_shared/helm///helm.mk:196: warning: undefined variable `mktemp -d'
make/_shared/helm///helm.mk:196: warning: undefined variable `mktemp -d'
make/_shared/helm///helm.mk:196: warning: undefined variable `mktemp -d'
Usage: make helm-diff helm_chart_old_version=<version>
make: *** [helm-diff] Error 1
```

I guess it could be:


```suggestion
	$(eval export TMP_OLD=$(shell mktemp -d))
	$(eval export TMP_NEW=$(shell mktemp -d))
	$(eval export OUTPUT_DIR=$(shell mktemp -d))
```